crazyflies:
  - id: 1
    channel: 100
    initialPosition: [-4.5, 0.0, 0.0]
    type: default
